### ISSUE TITLE
Git information no longer shown in prompt

### ISSUE THREAD

**Issue by MarcelRobitaille** [sindresorhus/pure#649]
#### System report (output of `prompt_pure_system_report`):

- Zsh: zsh 5.9 (x86_64-pc-linux-gnu) (/bin/zsh)
- Operating system: Linux (6.0.2-zen1-1-zen #1 ZEN SMP PREEMPT_DYNAMIC Sat, 15 Oct 2022 14:00:51 +0000 x86_64 GNU/Linux)
- Terminal program: tmux (3.3a)
- Tmux: yes
- Git: git version 2.38.1
- Pure state:
    - username: `''`
    - prompt: `❯`
    - version: `1.20.4`
- zsh-async version: `1.8.3`
- PROMPT: `typeset -g PROMPT=$'%F{${prompt_pure_colors[path]}}%~%f
%}%(12V.%F{$prompt_pure_colors[virtualenv]}%12v%f .)%(?.%F{$prompt_pure_colors[prompt:success]}.%F{$prompt_pure_colors[prompt:error]})${prompt_pure_state[prompt]}%f '`
- Colors: `typeset -g -A prompt_pure_colors=( [execution_time]=yellow [git:action]=yellow [git:arrow]=cyan [git:branch]=242 [git:branch:cached]=red [git:dirty]=218 [git:stash]=cyan [host]=242 [path]=blue [prompt:continuation]=242 [prompt:error]=red [prompt:success]=magenta [suspended_jobs]=red [user]=242 [user:root]=default [virtualenv]=242 )`
- TERM: `export TERM=xterm-256color`
- Virtualenv: `export VIRTUAL_ENV_DISABLE_PROMPT=12`
- Conda: `export CONDA_CHANGEPS1=no`
- Detected frameworks: Oh My Zsh, Zplug
    - Oh My Zsh:
        - Plugins: pass

#### Other information

- Pure version: 47c0c881f0e7cfdb5eaccd335f52ad17b897c060
- ZSH version: 5.9
- Terminal program & version: Alacritty
- Operating system: Arch Linux 6.0.2
- ZSH framework: oh-my-zsh
- Git version: 2.38.1
- Oh-my-zsh version: a7d910c3a61d8599f748a8ddae59ecdd9424022a

#### I have:

- [x] Tested with another terminal program and can reproduce the issue: Konsole
- [x] Followed the [integration](https://github.com/sindresorhus/pure#integration) instructions for my framework

### Problem description
Recently, the Git information like branch, dirty status, stash status has disappeared from my prompt. I did not update Pure, but it's possible Git or ZSH updated through a system update.

### Reproduction steps

1. Open a folder with a `.git`
2. Notice that the branch and dirty status are not shown

### My `.zshrc`:

```shell
fpath+=$HOME/.zsh/pure

autoload -U promptinit; promptinit
prompt pure
```

**Comment by mafredri** [comment]
It seems like you’ve updated Oh My Zsh 3 days ago. And I’d expect OMZ usage to be reflected in your `.zshrc`, but it’s not? Perhaps it’s related to that?

**Comment by MarcelRobitaille** [comment]
> It seems like you’ve updated Oh My Zsh 3 days ago

Actually, I found out today that I was using a version of oh my zsh from 2017. I pulled the latest version and reproduced the issue, so that's why I put that version in the issue.

> And I’d expect OMZ usage to be reflected in your .zshrc, but it’s not? Perhaps it’s related to that?

I was trying to create the most minimal `.zshrc` that could reproduce the issue. But, just now I commented back in the oh my zsh stuff... and it's working. I'll try to investigate and narrow down what in my config is causing the problem.
